packages/components/src/core/DropdownMenu/index.tsx

```diff
@@ -104,41 +102,61 @@ const DropdownMenu = <
         {
           name: "offset",
           options: {
-            offset: [0, 8],
+            offset: [0, theme.app?.spacing.s],
           },
         },
       ]}
       open={open}
       anchorEl={anchorEl}
-      {...PopperBaseProps}
       placement={PopperPlacement}
+      {...PopperBaseProps}
+      sx={{
+        ...PopperBaseProps?.sx,
+        padding: isMultiColumn
+          ? search || title
+            ? multiColumnWithSearchOrTitlePadding
+            : multiColumnNoSearchOrTitlePadding
+          : singleColumnPadding,
+        width:
+          (options &&
+            options[0] &&
+            Object.prototype.hasOwnProperty.call(options[0], "options")) ||
+          width < 160
+            ? "auto"
+            : width,
+      }}
     >
       <ClickAwayListener onClickAway={onClickAway} {...ClickAwayListenerProps}>
-        <div>
+        <StyledDropdownMenuAutocompleteWrapper
+          search={search}
+          title={title}
+          isMultiColumn={isMultiColumn}
+        >
           {title && (
             <StyledHeaderTitle search={search}>{title}</StyledHeaderTitle>
           )}
 
-          <Autocomplete
-            label={label}
-            search={search}
-            InputBaseProps={InputBaseProps}
-            PaperComponent={useCallback(
-              (paperComponentProps: PaperProps) => (
-                <PaperComponent
-                  search={search}
-                  title={title}
-                  {...paperComponentProps}
-                />
-              ),
-              [PaperComponent, search, title]
-            )}
-            open={open}
-            {...rest}
-            PopperComponent={defaultPopperComponent}
-          />
+          {anchorEl && (
+            <StyledAutocomplete
+              label={label}
+              search={search}
+              title={title}
+              open={open}
+              options={options}
+              InputBaseProps={{
+                ...InputBaseProps,
+                onClick: noop,
+              }}
+              PopperBaseProps={{
+                disablePortal: true,
+              }}
```

Comment:
this can be extracted out as a constant!

```ts
const PopperBaseProps = {
  disablePortal: true,
}

...

PopperBaseProps={PopperBaseProps}
```